htcanalyze/resource.py

```diff
@@ -0,0 +1,133 @@
+"""Class to define HTCondor Joblog resources."""
+
+from typing import List
+from numpy import nan_to_num as ntn
+
+
+class Resource:
+    """
+    Class of one single HTCondor-JobLog resource.
+
+    One file contains usually multiple resources,
+    so that nested lists represent a collection of job resources
+    """
+
+    level_colors = {'error': 'red', 'warning': 'yellow',
+                    'light_warning': 'yellow2', 'normal': 'green'}
+
+    def __init__(self,
+                 name: str,
+                 usage: float,
+                 requested: float,
+                 allocated: float,
+                 warning_level: str = None):
+        # naming in __init__ arguments should be
+        # consistent with names in the resources dictionary
```

Comment:
Minor nitpick, add 2 spaces before "consistent", which makes it a continuation of the comment line before it.

Reply:
Easy fix here :)